pkg/packagekit/updates.jsx

```diff
@@ -911,11 +871,9 @@ class OsUpdates extends React.Component {
                             <span className="fa fa-check" />
                         </div>
                         <p>{_("System is up to date")}</p>
-
-                        { this.state.history && !this.state.autoUpdatesEnabled
-                            ? <div className="flow-list-blank-slate"><UpdateHistory history={this.state.history} limit="1" /></div>
-                            : null }
                     </div>
+
+                    <History packagekit={this.state.autoUpdatesEnabled ? [] /* not tracked by PackageKit */ : this.state.history} />
```

Comment:
Not sure if this is regression but now if you have automatic updated enabled and you refresh the packakit page you see for a second the history component.
This would not be the case if you did this.state autoUpdatesEnabled && <History ..

Reply:
Well spotted, thanks! This indeed looks ugly. `autoUpdatesEnabled && <History` is indeed a better solution for *this* PR. It may be too strong for #11201, but we need to think how we want to present the history in the presence of automatic updates. But let's do that in #11201, and I'll fix this PR.

Reply:
Actually, the "flashing" happens if the detection whether autoupdates are enabled isn't finished yet, thus `autoUpdatesEnabled` is `null`. That's simple to fix.